### PR TITLE
Ctest/Drivers: Update to lightsaber

### PR DESCRIPTION
**Pull request by csiefer2**
Update to lightsaber driver since something decided to hate csh

### CODE REVIEW

**Review by cgcgcg — APPROVED** [review]

File: cmake/ctest/drivers/lightsaber/cron_driver.sh [hunk 1]
```diff
@@ -8,36 +8,36 @@ echo
...
-setenv TDD_HTTP_PROXY "http://wwwproxy.sandia.gov:80"
-setenv TDD_HTTPS_PROXY "https://wwwproxy.sandia.gov:80"
-setenv http_proxy "http://wwwproxy.sandia.gov:80"
-setenv https_proxy "https://wwwproxy.sandia.gov:80"
-setenv TDD_FORCE_CMAKE_INSTALL 1
-setenv TDD_DEBUG_VERBOSE 1
+export TDD_HTTP_PROXY="http://wwwproxy.sandia.gov:80"
+export TDD_HTTPS_PROXY="https://wwwproxy.sandia.gov:80"
+export http_proxy="http://wwwproxy.sandia.gov:80"
+export https_proxy="https://wwwproxy.sandia.gov:80"
```
Comment (jhux2) [inline cmake/ctest/drivers/lightsaber/cron_driver.sh:26]: `wwwproxy.sandia.gov` should be `proxy.sandia.gov`.

**Review by jhux2 — CHANGES_REQUESTED** [review]

File: cmake/ctest/drivers/lightsaber/cron_driver.sh (hunk 1 above)
Comment (csiefer2) [reply cmake/ctest/drivers/lightsaber/cron_driver.sh:26]: You're right, but since we don't actually use external network connections, it doesn't matter.

Comment (jhux2) [reply cmake/ctest/drivers/lightsaber/cron_driver.sh:26]: just delete?

**Review by jhux2 — APPROVED** [review]
I laughed, I cried

**Pull request merged**